### PR TITLE
finished rating workflow

### PR DESCRIPTION
**Pull request by aaronyoo**

### CODE REVIEW

File: swiper/app/controllers/listings_controller.rb [hunk 1]
```diff
@@ -172,6 +172,22 @@ def reserve
...
+
+        params[:rating] = params[:rating].to_i
+        if params[:rating] > 5 || params[:rating] < 0
+            redirect_to listing_path(@listing, :id => params[:id]), :flash => { :error => "You submitted an invalid rating value!" } and return
+        else
+            puts @seller.rating
+            if @seller.rating == nil
+                new_rating = params[:rating]
+            else
+                new_rating = @seller.rating + params[:rating]
```
Comment (indiandennis) [inline swiper/app/controllers/listings_controller.rb]: shouldn't this be + params[:rating] / rating_count?

Comment (aaronyoo) [reply swiper/app/controllers/listings_controller.rb]: Oh, are we keeping a running average? That makes sense why its a float

Comment (indiandennis) [reply swiper/app/controllers/listings_controller.rb]: or more accurately new rating = (@seller.rating + params[:rating]) / (rating_count + 1)

Comment (aaronyoo) [reply swiper/app/controllers/listings_controller.rb]: No, that value would quickly converge to zero

**Review by indiandennis — COMMENTED** [review]
doesn't this rating continuously increase now? if the current rating is 4 and the user rates them a 2, it will still go up. It should probably be something like seller.rating + (params[:rating] - @seller.rating) / (@seller.rating_count + 1)

**Pull request merged**